File: examples/cpp/vrp/cg/path.cpp
// Copyright (c) 2025 Laboratory for Combinatorial Optimization in Real-time Environment.
// All rights reserved.

#include "path.hpp"

Path::Path(size_t id, double cost, const std::vector<size_t>& visited_nodes)
    : id(id), cost(cost), visited_nodes(visited_nodes) {}

File: examples/python/vrp/cg/path.py
#  Copyright (c) 2025 Laboratory for Combinatorial Optimization in Real-time Environment.
#  All rights reserved.


class Path:
    def __init__(self, path_id, cost, visited_nodes):
        self.id = path_id
        self.cost = cost
        self.visited_nodes = list(visited_nodes)
